neat-form-core/src/main/java/com/nerdstone/neatformcore/views/builders/MultiChoiceCheckBoxViewBuilder.kt

```diff
@@ -41,6 +45,20 @@ class MultiChoiceCheckBoxViewBuilder(override val nFormView: NFormView) : ViewBu
                     ViewUtils.addViewLabel(attribute.toPair(), multiChoiceCheckBox)
                 )
             }
+
+            MultiChoiceCheckBoxProperties.LABEL_TEXT_SIZE.name -> {
+                try {
+                    multiChoiceCheckBox.findViewById<TextView>(R.id.labelTextView).apply {
+                        textSize = attribute.value.toString().toFloat()
+                    }
+                }catch (e:NullPointerException){
+                    Timber.e(e)
+                }
+            }
```

Comment:
I do not think the try catch is necessary. You can use save **elvis operator** for null check without crashing the app. Something like this:

` multiChoiceCheckBox.findViewById<TextView>(R.id.labelTextView)?.apply {
                    textSize = attribute.value.toString().toFloat()
                }
`